spec/controllers/shipments_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+require 'pry'
+
+RSpec.describe ShipmentsController, type: :controller do
+  describe "POST 'estimate'" do
+    let(:params) do
+      {origin: {country: 'US',
+                 state: 'CA',
+                 city: 'Beverly Hills',
+                 zip: '90210'},
+
+      destination: {country: 'CA',
+                    province: 'ON',
+                    city: 'Ottawa',
+                    postal_code: 'K1P 1J1'},
+      package: { weight: 100,
+                length: 93,
```

Comment:
Indent the following one more space to line up with weight. On that note I like how you have space out destination, it is very easy to read.